Settlement/index.ts

```diff
@@ -1,13 +1,20 @@
-import { Conversion } from "./Conversion"
+import * as conversion from "./conversion"
 import { Fee as SettlementFee } from "./Fee"
+import { Metabase as SettlementMetabase } from "./Metabase"
 import { Settlement as modelSettlement } from "./Settlement"
 import { Transaction as SettlementTransaction } from "./Transaction"
 import { V1 as SettlementV1 } from "./V1"
 
 export type Settlement = modelSettlement
 export namespace Settlement {
-	export const toCsv = Conversion.toCsv
-	export const toDetailedCsv = Conversion.toDetailedCsv
+	export function toCsv(
+		settlements: modelSettlement[],
+		role?: "merchant" | "agent" | "master" | undefined,
+		costPlus?: true | undefined
```

Comment:
`undefined` is redundant
```suggestion
		role?: "merchant" | "agent" | "master",
		costPlus?: true
```

Reply:
Done